client/src/pages/admin/disciplines.tsx

```diff
@@ -0,0 +1,6 @@
+import { getServerSideProps } from '../../modules/Discipline/data/getServerSideProps';
+import { DisciplinePage } from '../../modules/Discipline/pages/DisciplinePage';
+import withSession from '../../components/withSession';
+
+export { getServerSideProps };
+export default withSession(DisciplinePage);
```

Comment:
there is intention to remove `withSession` and use SessionContext/Provider.

Please see https://github.com/rolling-scopes/rsschool-app/blob/master/client/src/pages/course/mentor/students.tsx#L8

Reply:
https://github.com/rolling-scopes/rsschool-app/blob/master/client/src/modules/Mentor/pages/Students/index.tsx#L14